nomenclature/config.py

```diff
@@ -126,12 +181,22 @@ def repos(self) -> dict[str, str]:
         }
 
 
+class MappingRepository(BaseModel):
+    name: str
```

Comment:
Das The mapping will also have to inherit the region-filters, right? Otherwise, a model could map to a region that is not included in the **DataStructureDefinition**.

Reply:
That's already taken care of here: https://github.com/phackstock/nomenclature/blob/27878178da182a48b85b524e857d2c80688b644a/nomenclature/processor/region.py#L485-L496. The external mapping repository contents are parsed like any other mapping file and are therefore checked for region consistency.